.gitignore

```diff
@@ -3,4 +3,4 @@ node_modules/
 .fusion/
 .nyc_output/
 yarn-error.log
-
+.idea/
```

Comment:
Not sure if we want to include ide-specific files in our gitignore files - might be better to add this to your user directory? I believe you can have your own files listed in a global `~/.gitignore` file which would work for all projects.

Reply:
removed